### ISSUE TITLE
Number of model parameters.

### ISSUE THREAD

**Issue by Demirrr** [uma-pi1/kge#142]
Hey,

thank you for great work! I was wondering whether the number of parameters of models --including RESCAL, ComplEx and ConvE have been reported in somewhere.

Cheers,
Caglar Demir

**Comment by rgemulla** [comment]
It's printed and logged whenever you train a model. For the published models, run

```sh
kge dump checkpoint fb15k-237-rescal.pt --keys valid_trace | grep num_parameters | head -1
```

to extract it. Is this what you mean?

**Comment by Demirrr** [comment]
I am afraid something is incomplete.

1. All steps in [quick-start](https://github.com/uma-pi1/kge#quick-start) are completed, i.e., the script of toy example was successfully executed.

2. Executing `kge dump checkpoint fb15k-237-rescal.pt --keys valid_trace | grep num_parameters | head -1` throwed the following exception `Exception: Could not find checkpoint in fb15k-237-rescal.pt`

My main goal is to fully make use of the results reported in "YOU CAN TEACH AN OLD DOG NEW TRICKS! ON TRAINING KNOWLEDGE GRAPH EMBEDDINGS". More specifically, I am interesting in the number of trainable parameters for all models in the Table2. Apart from this, I was wondering whether you could name one or two frameworks that remove unseen entities from the test set.

Cheers

**Comment by rgemulla** [comment]
You need to download the respective checkpoint from the [KGE homepage](https://github.com/uma-pi1/kge) first. The checkpoint links are in the "pretrained model" column.

**Comment by Demirrr** [comment]
It worked like a charm. Thanks!
